common/src/main/java/com/wynntils/models/statuseffects/type/StatusEffect.java

```diff
@@ -76,6 +85,18 @@ public StyledText asString() {
         return fullName;
     }
 
+    public StyledText getModifierSuffix() {
+        return this.modifierSuffix;
+    }
+
+    public boolean hasModifierValue() {
+        return this.modifierValue.isPresent();
+    }
+
+    public double getModifierValue() {
+        return this.modifierValue.get();
```

Comment:
This is so backwards.. The field should be a double and this method should return optionals. Optional fields should not exist, and you just call .get without checking if it exists here.

Reply:
What? I dont know how you want to do it, just do as i said.. optional fields shouldnt exist

Reply:
It would get rid of the optional field and not require storing a boolean for if there is no modifier for the effect

Reply:
You can just use the boxes Double class to allow null